File: odemapster-r2rml-rdb/src/main/java/es/upm/fi/dia/oeg/obdi/wrapper/r2rml/rdb/model/R2RMLSQLQuery.java
package es.upm.fi.dia.oeg.obdi.wrapper.r2rml.rdb.model;

import es.upm.fi.dia.oeg.obdi.core.model.AbstractConceptMapping;
import es.upm.fi.dia.oeg.obdi.core.sql.SQLFromItem.LogicalTableType;


public class R2RMLSQLQuery extends R2RMLLogicalTable {
	private String sqlQuery;
	
	public R2RMLSQLQuery(String sqlQuery) {
		this.sqlQuery = sqlQuery;
		super.logicalTableType = LogicalTableType.QUERY_STRING;
	}

	@Override
	public String getValue() {
		return this.sqlQuery;
	}
}

File: morph-r2rml/src/main/scala/es/upm/dia/fi/oeg/morph/r2rml/model/R2RMLSQLQuery.scala
package es.upm.dia.fi.oeg.morph.r2rml.model

import es.upm.fi.dia.oeg.morph.base.Constants


class R2RMLSQLQuery(sqlQuery:String ) 
extends R2RMLLogicalTable(Constants.LogicalTableType.QUERY_STRING) {

	override def getValue() : String = { this.sqlQuery; }
}